qiskit/transpiler/target.py

```diff
@@ -373,7 +381,15 @@ def add_instruction(self, instruction, properties=None, name=None):
         self._instruction_schedule_map = None
 
     def update_instruction_properties(self, instruction, qargs, properties):
-        """Update the property object for an instruction qarg pair already in the Target
+        """Update the property object for an instruction qarg pair already in the Target.
+
+        For ease of access, a user is able to obtain the mapping between an instruction's
+        applicable qargs and its instruction properties via indexing (using ``__getitem__``),
```

Comment:
Indexing isn't the correct term here, it's a mapping from name to the inner dict:

```suggestion
        applicable qargs and its instruction properties via the mapping protocol (using ``__getitem__``),
```